bin/lint

```diff
@@ -72,7 +72,7 @@ copyright_files=$(grep -vE \
 # Only binary files are permitted to omit a trailing newline. If you're here to
 # exclude a text file that is missing its trailing newline, like an SVG, add
 # a trailing newline to the text file instead.
-newline_files=$(grep -vE '\.(png|jpe?g|pb|avro|ico|so)$' <<< "$files")
+newline_files=$(grep -vE '\.(excalidraw|png|jpe?g|pb|avro|ico|so)$' <<< "$files")
```

Comment:
@benesch: this violates the comment above, but if we don't this every time somebody changes the source `~.excalidraw` asset, they will also have to manually add the endline at the end of the file. 

Not sure about the benefits of doing that rather than just adding it to the exclusion list.